### DISCUSSION TITLE
How to avoid parsing `.env` file for running tests and keep only `.env.test`

### DISCUSSION THREAD

**Discussion by cervebar** [goatandsheep/react-native-dotenv#479] [Q&A]
We are using `.env.test` file for our (jest) unit tests and `.env` for production.

```
MY_ENV="production"
```
but for test we want it empty:
```
MY_ENV=""
```

However when we add there empty value this way, the `dotenv` will probably consider it as undefined and take it from `.env` file.
This is causing tests behave differently on different environments (CI vs local dev) as we have different `.env` files but allways same `.env.test`.

How to prevent this behaviour?

**Comment by goatandsheep** [comment]
could `.env.test` and `.env.production` work?
